pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1518,14 +1527,13 @@ func TestPriorityQueue_MoveAllToActiveOrBackoffQueueWithQueueingHint(t *testing.
 					QueueingHintFn: test.hint,
 				},
 			}
-			test.podInfo.UnschedulablePlugins = sets.New("foo")
```

Comment:
This was wrong as each test case sets UnschedulablePlugins (or PendingPlugins) as necessary.

Reply:
If possible, these can also be a follow up to avoid potential code conflicts, I mean we 're intent to cherry-pick, right?

Reply:
Good point.

Reply:
Ah, wait. But, actually we have to remove it in this PR to test a scenario when test.podInfo.UnschedulablePlugins includes the scheduling gate (`"QueueHintFunction is not called when Pod is gated by SchedulingGates plugin"`).